src/Kmd.Logic.Gateway.Automation/PreValidation/EntityPreValidationBase.cs

```diff
@@ -29,29 +28,43 @@ public EntityPreValidationBase(string folderPath)
 
         public List<PublishResult> ValidationResults { get; }
 
-        protected bool ValidateFile(GatewayFileType fileType, string path, string entityName, string propName)
+        protected void ValidateFile(FileType fileType, string path, string entityName, string propName)
```

Comment:
return bool is good, yes I know return type (bool)  is not used, still thinking to have validation status successes or failed

Reply:
But why?

Reply:
ok fixed.